llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -285,9 +286,24 @@ int main(int argc, char *argv[]) {
     MSG("\n");
   }
 
-  if (!ComputeStats.empty())
-    computeStats(*CAS, SummaryIDs);
+  if (!ComputeStats.empty()) {
+    bool PrintToStdout = false;
+    if (StringRef(ComputeStats) == "-")
+      PrintToStdout = true;
 
+    llvm::Optional<raw_fd_ostream> CsvStream;
```

Comment:
I think there's a `using namespace llvm` at the top of this file, so you can just say `Optional` here.